stories/layout-grid/layout-grid.stories.js

```diff
@@ -290,7 +290,93 @@ offset.parameters = {
     }
 };
 
-export const noGap = () => `<div class="fd-container fd-container--no-gap">
+export const noHorizontalGap = () => `<div class="fd-container fd-container--no-horizontal-gap">
```

Comment:
Can't we move it to the `fd-row` level?

Reply:
As discussed, we can't due to negative margines